### PR TITLE
Set readOnly false for credit memo tax object items

### PR DESCRIPTION
**Pull request by nanobash**
## Summary

[Preview](https://preview.redoc.ly/rebilly-core/remove-readonly-on-credit-memo-tax-object/tag/Credit-memos/operation/PostCreditMemo#!path=items/tax&t=request)

- [x] Set readOnly false for credit memo tax object items

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/CreditMemos/CommonCreditMemo.yaml [hunk 1]
```diff
@@ -135,6 +135,37 @@ properties:
...
+                  properties:
+                    country:
+                      readOnly: false
+                    state:
+                      readOnly: false
+                    county:
+                      readOnly: false
+                    city:
+                      readOnly: false
             - $ref: ../Invoices/InvoiceTaxItem.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/CreditMemos/CommonCreditMemo.yaml:169]: Perhaps better to introduce a CreditMemoTaxItem schema

Comment (nanobash) [reply openapi/components/schemas/CreditMemos/CommonCreditMemo.yaml]: Maybe, updated, please check, thanks

Comment (nanobash) [reply openapi/components/schemas/CreditMemos/CommonCreditMemo.yaml]: Updated as discussed in person, thanks

**Review by ashkarpetin — DISMISSED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Comment by nanobash** [comment]
@Rebilly/php please review

**Review by ninjasimon — APPROVED** [review]

**Pull request merged**